topics/admin/tutorials/monitoring/tutorial.md

```diff
@@ -780,7 +781,7 @@ You can run the playbook now, or wait until you have configured Telegraf below:
 >    +    privs: SELECT
 >    +    objs: ALL_IN_SCHEMA
 >    +
->     
```

Comment:
these whitespace changes are not ideal, they're in the commit blocks which may break the export process. Could you please restore the whitespace? I would recommend turning off any "remove trailing whitespace" plugin for the GTN repository as here it is meaningful in the admin diffs.